### PR TITLE
Add skip and pageable

### PR DESCRIPTION
**Pull request by cjavier70**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -24,7 +24,7 @@
 
   <groupId>com.github.krraghavan</groupId>
   <artifactId>mongodb-aggregate-query-support</artifactId>
-  <version>0.7.8</version>
+  <version>0.7.9</version>
```
Comment (krraghavan) [inline pom.xml]: Don't do this here.  I'll do it from the master branch.

File: src/main/java/com/cisco/mongodb/aggregate/support/query/AggregateQueryProvider.java [hunk 2]
```diff
@@ -297,18 +304,39 @@ private void createAggregateQuery() throws InvalidAggregationQueryException {
...
     }
   }
 
+  private void addToEndOfQuery(int pipelineCount, boolean outAnnotationPresent, boolean isPageable, Out out,
+                               String[] queries, Pageable pageable) {
+    int lastStage = pipelineCount - 1;
+    int skipStageForPageable = lastStage - 1;
+    int limitStageForPageable = lastStage;
+    //shift skipStageForPageable back one if client is performing an out
+    if (outAnnotationPresent) {
```
Comment (krraghavan) [inline src/main/java/com/cisco/mongodb/aggregate/support/query/AggregateQueryProvider.java]: Add some logger statements for debuggability.

**Review by krraghavan — CHANGES_REQUESTED** [review]
Just a few minor changes.  Can you please make them and resubmit.

File: pom.xml (hunk 1 above)
Comment (cjavier70) [reply pom.xml]: Done

File: src/main/java/com/cisco/mongodb/aggregate/support/query/AggregateQueryProvider.java (hunk 2 above)
Comment (cjavier70) [reply src/main/java/com/cisco/mongodb/aggregate/support/query/AggregateQueryProvider.java]: done

**Review by krraghavan — APPROVED** [review]

**Pull request merged**